src/astra_driver.cpp

```diff
@@ -729,6 +729,15 @@ sensor_msgs::msg::CameraInfo::SharedPtr AstraDriver::getDepthCameraInfo(int widt
 void AstraDriver::readConfigFromParameterServer()
 {
   depth_frame_id_ = std::string("openni_depth_optical_frame");
+
+  // Load the depth registration parameter, which may have been set before
+  //   driver initialization.
+  pnh_->get_parameter("depth_registration", depth_registration_);
+  if(depth_registration_)
+  {
+    std::cout << "Astra depth registration enabled" << std::endl;
```

Comment:
I think we generally prefer to use the ROS_ macros, so in this case, I think you'd want to use ROS_INFO().